### ISSUE TITLE
[BUG] "The option `opts' does not exist"

### ISSUE THREAD

**Issue by matthiasbeyer** [nix-community/nixvim#2509]
| Field        | Description |
|--------------|-------------|
| Nixpkgs      | `24.05` | 

- [x] I have read the [FAQ](https://nix-community.github.io/nixvim/user-guide/faq.html) and my bug is not listed there.

## Minimal, Reproducible Example (MRE)

I have the following in my flake:

```nix
# ... 
let
    nixvimModule = {
      pkgs = inputs.nixpkgs.legacyPackages."x86_64-linux";
      module = { ... }: {
        config.opts.number = true;
      };
    };

    nvim = inputs.nixvim.legacyPackages."x86_64-linux".makeNixvimWithModule
      nixvimModule;
in
  nvim
```

**Comment by MattSturgeon** [comment]
Yes, nixvim `main` and stable (`nixos-24.05`) are different versions of nixvim. Sometimes there are significant differences between the current unstable and stable.

In this case, `options` was renamed to `opts` [way back in March](https://github.com/nix-community/nixvim/pull/1324).

Nixvim's "stable" docs can be found at https://nix-community.github.io/nixvim/stable/ (note the trailing `/stable` in the URL).

You may also be interested in https://github.com/nix-community/nixvim/pull/2301, which allows you to use an unstable nixvim (with an unstable nixpkgs) _within_ a stable host (that uses a stable nixpkgs) by setting `programs.nixvim.nixpkgs.pkgs` to an unstable nixpkgs instance.

**Comment by MattSturgeon** [comment]
> In this case, `options` was renamed to `opts` [way back in March](https://github.com/nix-community/nixvim/pull/1324).

Although, that should be in both 24.05 and unstable, given march (03) was _before_ May when 24.05 was branched off...

**Comment by matthiasbeyer** [comment]
Yep, I found everything by now. This issue may serve as a good documentation for others stumbling over this.

I'll close it though, as my issue is solved here. Thanks!
